### PR TITLE
cmake.eclass EAPI7--

### PR DESCRIPTION
**Pull request by a17r**
### Please bump all future ebuild versions to EAPI-8. ###

### CODE REVIEW

**Comment by sbraz** [comment]
Thanks, I applied the znc commit in 8f1cce5259f44d283ce124d0ec04325cd345f94a.

**Review by ZeroChaos- — APPROVED** [review]

**Comment by mm1ke** [comment]
the `app-emulation/dynamips` drop is fine for me

**Review by chewi — APPROVED** [review]

**Review by xgqt — APPROVED** [review]
app-emacs/* lgtm

**Comment by slashbeast** [comment]
I would appreciate having a description and somewhat more informative merge request title. I will drop old libtorrent-rasterbar ebuilds in a moment.

**Comment by a17r** [comment]
I guess one can't please everyone.

**Comment by SoapGentoo** [comment]
for everyone I know, "EAPI-7--" is pretty much self-explanatory. And as always, if you want better wording, make a suggestion...

**Comment by mgorny** [comment]
> "EAPI-7--" is pretty much self-explanatory

Yes, 7-- gives 6.

**Comment by grobian** [comment]
uncrustify-0.75.1 dropped, thanks

**Comment by waebbl** [comment]
no objections dropping media-libs/ptex-2.3.2, thanks

**Comment by amadio** [comment]
Fine for net-fs/cvmfs.

**Comment by tamiko** [comment]
@a17r acked.

**Review by tamiko — APPROVED** [review]

**Comment by arkamar** [comment]
LGTM for `dev-libs/hyperscan` & `mail-filter/rspamd`

**Review by jkroonza — APPROVED** [review]
For net-misc/sipp.

File: profiles/package.mask [hunk 1]
```diff
@@ -33,6 +33,12 @@
 
 #--- END OF EXAMPLES ---
 
+# Andreas Sturmlechner <asturm@gentoo.org> (2023-06-18)
+# No revdeps, blocking cleanup of dev-libs/libfmt-8; bug #908651
+# Removal on 2023-07-18.
+sci-libs/rocSOLVER
+sci-libs/hipBLAS
```
Comment (heroxbd) [inline profiles/package.mask:40]: Fixed in https://bugs.gentoo.org/908651 . Please undo.

**Pull request merged**